style.css

```diff
@@ -621,6 +636,65 @@
   .nav-link {
     display: none;
   }
+
+  #menu- {
+    z-index: 1;
+    background-color: #3c3a39;
+    height: 720px;
+    display: none;
+    position: absolute;
+    width: 100%;
+  }
+
+  #closed {
+    float: right;
+    margin-right: 30%;
+    color: #67798e;
+    font-size: 20px;
+    margin-top: 62px;
+  }
```

Comment:
As suggested by the previous reviewer kindly position the close button properly to make the Figma design and create a good user experience. 
Your design:
<img width="324" alt="Screenshot 2023-04-10 at 20 11 34" src="https://user-images.githubusercontent.com/107192076/230990506-b14902f0-7a41-4510-bf8e-e69eeed71eda.png">


You can set it properly by adjusting the `margin-right: 8%` and the `margin-top: 20px` instead of what you have.